src/Nethermind/Nethermind.JsonRpc.Test/Modules/AdminModuleTests.cs

```diff
@@ -66,11 +66,12 @@ public void Test_node_info()
             settings.Converters = EthereumJsonSerializer.CommonConverters.ToList();
             
             NodeInfo nodeInfo = ((JObject) response.Result).ToObject<NodeInfo>(JsonSerializer.Create(settings));
-            nodeInfo.Enode.Should().Be(_enodeString);
+            var enode = new Enode(EnodeString);
+            nodeInfo.Enode.Should().Be(enode.ToString());
             nodeInfo.Id.Should().Be("ae3623ef35c06ab49e9ae4b9f5a2b0f1983c28f85de1ccc98e2174333fdbdf1f");
-            nodeInfo.Ip.Should().Be("127.0.0.1");
+            nodeInfo.Ip.Should().Be(enode.HostIp.ToString());
```

Comment:
instead of adding new tests we replaced existing ones -need to tets both IP and DNS